util/authorization.py

```diff
@@ -6,9 +6,12 @@
 
 
 def get_authorization(authorization: str = Header(default=None)):
-    """
-    Extract authorization details out of the token
-    """
+    """Middleware to extract authorization details out of the token"""
     token = authorization.split(" ")[1]
```

Comment:
Will the authorization always in every-case have something in [1] index?

Reply:
Yes, without exception. The authorization always comes in as a string in the format `Bearer {some-token}`, this just cuts out the "Bearer" part